config/channel/consolidated/configmaps/kafka-config.yaml

```diff
@@ -13,13 +13,20 @@
 # limitations under the License.
 
 apiVersion: v1
+data:
+  version: 1.0.0
+  # eventing-kafka.kafka.brokers: Replace this with the URLs for your kafka cluster,
+  #   which is in the format of my-cluster-kafka-bootstrap.my-kafka-namespace:9092.
+  # eventing-kafka.kafka.authSecretName: name-of-your-secret-for-kafka-auth
+  # eventing-kafka.kafka.authSecretNamespace: namespace-of-your-secret-for-kafka-auth
```

Comment:
@eric-sap can we have some doc update for these changes on the docs repo:
https://github.com/knative/docs/tree/mkdocs/docs/eventing/samples/kafka/channel#authentication-against-an-apache-kafka

Reply:
Absolutely; I will get on that.  Thank you for the pointer.